ui/src/app/routing/guards.tsx

```diff
@@ -39,8 +39,7 @@ export const expiredGuard: Guard = {
   redirectPath: '/access-renewal'
 };
 
-export const adminLockedGuard = (): Guard => {
-  const {ns, wsid} = useParams<MatchParams>();
```

Comment:
violates "Only Call Hooks from React Functions"

alt solution: make this a custom hook.  That didn't seem worth the trouble here.